### DISCUSSION TITLE
Use loadFromJSONto restore without clearing other contents on the canvas

### DISCUSSION THREAD

**Discussion by Loverz55** [fabricjs/fabric.js#8533] [Q&A] [answered]
I used loadFromJSON many times, but I want to keep the last restored

**Comment by Loverz55** [comment]
I wanted to restore multiple Jsons in different methods, but loadFromJson will clean the canvas

**Comment by iocoker** [comment, marked as answer]
I think what you need to do is use `loadFromJson` first time only, and subsequently, you want to parse the objects and place them on the canvas yourself.

You can get the array of objects and use `fabric.utils.enlivenObjects` to parse the objects into fabric Objects and add them to the canvas.
[http://fabricjs.com/docs/fabric.util.html#.enlivenObjects](http://fabricjs.com/docs/fabric.util.html#.enlivenObjects)
